src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/MapperTest.cs

```diff
@@ -7,6 +8,9 @@ namespace DevAdventCalendarCompetition.Tests
     public class MapperTest
     {
         [Fact]
+#pragma warning disable CA1041 // Provide ObsoleteAttribute message
```

Comment:
please add a message [Obsolete("This method should be removed")] in this attribute and remove pragma warning